src/test/java/com/solvd/laba/carina/SignupTest.java

```diff
@@ -0,0 +1,52 @@
+package com.solvd.laba.carina;
+
+import com.solvd.laba.carina.products.gui.pages.AccountCreatedPage;
+import com.solvd.laba.carina.products.gui.pages.HomePage;
+import com.solvd.laba.carina.products.gui.pages.SignupLoginPage;
+import com.solvd.laba.carina.products.gui.pages.SignupPage;
+import com.solvd.laba.carina.products.utils.TestUtil;
+import com.zebrunner.carina.core.IAbstractTest;
+import com.zebrunner.carina.core.registrar.ownership.MethodOwner;
+import com.zebrunner.carina.utils.R;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class SignupTest implements IAbstractTest {
+
+    @Test
+    @MethodOwner(owner = "hhan")
+    public void signupTest() {
+        String name = R.TESTDATA.get("name");
+        String email = R.TESTDATA.get("signupEmail");
```

Comment:
is this test repeatable with using  the same email?

Reply:
I have created a method to generate random emails.